webcompat/templates/layout.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>webcompat.com</title>
+  <title>{% block title %}{% endblock %}webcompat.com</title>
```

Comment:
ah good point. Probably we could have a title such as `Web Compatibility | webcompat.com`

Let's see. on Duckduckgo
![capture d ecran 2017-03-13 a 06 49 19](https://cloud.githubusercontent.com/assets/505230/23843039/5662a4b2-07b9-11e7-94c5-58614d1bea83.png)

on Google

![capture d ecran 2017-03-13 a 06 51 12](https://cloud.githubusercontent.com/assets/505230/23843062/808bbe0e-07b9-11e7-9446-0ad9725a8587.png)

On Bing

![capture d ecran 2017-03-13 a 06 52 26](https://cloud.githubusercontent.com/assets/505230/23843091/a87c36fa-07b9-11e7-91f7-fb6b14f8579a.png)

Yeah I would probably add `Web Compatibility | webcompat.com` for the home page.